extmod/modbluetooth.c

```diff
@@ -918,11 +935,15 @@ STATIC bool enqueue_irq(mp_obj_bluetooth_ble_t *o, size_t len, uint16_t event) {
     return true;
 }
 
-STATIC void schedule_ringbuf(void) {
+// Must not hold the atomic section.
```

Comment:
shouldn't this be the opposite, "Must hold the Bluetooth atomic lock upon entry to this function."?

Reply:
Done.